### ISSUE TITLE
Feat: Conversion of png images to vector images

### ISSUE THREAD

**Issue by Bhavnaharitsa** [openMF/mobile-wallet#709]
**Summary:**

Till now, all our images are png based. For such heavy projects its be good if we converted png images to vector images.

**Comment by Bhavnaharitsa** [comment]
@naman14 Your thoughts sir?

**Comment by HaripriyaB** [comment]
Pictures you mean to say the profile pic or anyother??

**Comment by Bhavnaharitsa** [comment]
@HaripriyaB No, how can we convert that!! Its what the user uploads, but the png images as clearly given in issue title

**Comment by HaripriyaB** [comment]
So what images are to be converted?

**Comment by Bhavnaharitsa** [comment]
@HaripriyaB The images already present in the app.

**Comment by HaripriyaB** [comment]
Can I work on it? Pls guide me through..

**Comment by Bhavnaharitsa** [comment]
@HaripriyaB I have taken up the issue and am working on it. I'm sorry, but you can look for other issues :)

**Comment by therajanmaurya** [comment]
Fixed in compose implementation
